### DISCUSSION TITLE
Transaction info on roulete is blank on polygonscan

### DISCUSSION THREAD

**Discussion by bf-mike-mkt** [betfinio/public#77] [Bugs and Reports]
I go to Roulette, all bets, and click on any magnifier glass icon, then I click to BET ID link. When opened on polygonscan it will tell me there is no transaction. Should it work like this? 

https://polygonscan.com/address/0x8a37033905a8442bbd9d103ebe3a4b035264f4d2

![image](https://github.com/user-attachments/assets/1972aa1e-5fb2-478e-ad30-91d54c93f751)

**Comment by bf-monster** [comment]
Hey, check again, link is ok
